packages/synapse-react-client/src/template_style/_form.scss

```diff
@@ -16,7 +16,7 @@
 
 .checkbox,
 .radio {
-  padding-left: 6px;
+  padding-left: 0px;
```

Comment:
So checkboxes can be left-aligned with their container

Reply:
https://sagebionetworks.jira.com/browse/PORTALS-2732 may need to be revisited

Reply:
Think I have a full solution here: [`b8a6171` (#475)](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/475/commits/b8a6171c043ebb35dec9a4b8a0a9fefa096cc766)